django/contrib/postgres/constraints.py

```diff
@@ -155,12 +155,15 @@ def __eq__(self, other):
         return super().__eq__(other)
 
     def __repr__(self):
-        return '<%s: index_type=%s, expressions=%s%s%s%s%s>' % (
+        return "<%s: index_type='%s' name=%r expressions=%s%s%s%s%s>" % (
             self.__class__.__qualname__,
             self.index_type,
-            self.expressions,
-            '' if self.condition is None else ', condition=%s' % self.condition,
-            '' if self.deferrable is None else ', deferrable=%s' % self.deferrable,
-            '' if not self.include else ', include=%s' % repr(self.include),
-            '' if not self.opclasses else ', opclasses=%s' % repr(self.opclasses),
+            self.name,
+            "'%s'" % ', '.join([
+                str(expression) for expression in self.expressions
+            ]),
```

Comment:
After checking changes in tests I think using `repr()` for `expressions`, `fields`, `include`, and `opclasses` makes a representation more readable (it's easy to miss a comma and don't notice the list of values), e.g.
```python
' expressions=%s' % repr(self.expressions)
' fields=%s' % repr(self.fields)
' opclasses=%s' % repr(self.opclasses)
' include=%s' % repr(self.include)
```